runtime/vm/jvminit.c

```diff
@@ -1192,19 +1192,28 @@ static void loadDLL(void* dllLoadInfo, void* userDataTemp) {
  *  Every library used by the VM should have an entry here, except for any user Xruns.
  */
 static J9Pool *
-initializeDllLoadTable(J9PortLibrary *portLibrary, J9VMInitArgs* j9vm_args, UDATA verboseFlags)
+initializeDllLoadTable(J9PortLibrary *portLibrary, J9VMInitArgs* j9vm_args, UDATA verboseFlags, J9JavaVM *vm)
 {
 	J9Pool *returnVal = pool_new(sizeof(J9VMDllLoadInfo),  0, 0, 0, J9_GET_CALLSITE(), OMRMEM_CATEGORY_VM, POOL_FOR_PORT(portLibrary));
 	IDATA i;
 	char* testString, *options;
 	J9VMDllLoadInfo* newEntry;
 	char dllNameBuffer[SMALL_STRING_BUF_SIZE];			/* Plenty big enough - needs to be at least DLLNAME_LEN */
+	char *gcDLLName = J9_GC_DLL_NAME;
+	char *gccheckDLLName = J9_CHECK_GC_DLL_NAME;
```

Comment:
The types need to be `const char *`.

Reply:
In this file, I get `error: passing 'const char *' to parameter of type 'char *' discards qualifiers` since the `createLoadInfo()` function where those names are used expects `char *`

Reply:
It appears that the signature of `createLoadInfo` can easily be fixed (so the name parameter is of type `const char *`).